eng/versioning/version_data_java_files.txt

```diff
@@ -3,5 +3,4 @@
 # has been fixed this file and the update logic in update_versions.py should no longer be necessary.
 
 # The file format here should be the relative path from the root of the azure-sdk-for-java
-sdk/eventhubs/microsoft-azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/ClientConstants.java
 sdk/storage/microsoft-azure-storage-blob/src/main/java/com/microsoft/azure/storage/blob/Constants.java
```

Comment:
What is still using this? We've removed this from our source code.

Reply:
Good catch. I'm also not convinced this is useful anymore.. the comment in `update_versions.py` points to #7106 which has been closed for a while. @conniey Should we do something about this and get rid of this text file entirely at some point?